pattern-library/js/select-replace.js

```diff
@@ -60,16 +57,21 @@ define([
 
                     $el.replaceWith([
                         '<div class="' + variables.wrapperClass + '">',
-                            '<select class="' + replaced[0].className + ' is-replaced" id="' + replaced[0].id + '" name="' + replaced[0].name + '" ' + disabled + '>' + replaced[0].innerHTML + '</select>',
-                            '<span class="' + variables.customClass + ' ' + statuses.join(' ') + '" aria-hidden="true">',
-                                '<span class="' + variables.valueClass + '">' + CustomSelectReplacement.setInitialText($el) + '</span>',
-                                '<span class="icon-fallback-glyph">',
-                                    '<span class="icon ' + variables.iconClass + '" aria-hidden="true"></span>',
-                                    '<span class="text">Down arrow</span>',
-                                '</span>',
-                            '</span>',
+                        '<select class="' + replaced[0].className + ' is-replaced" id="' + replaced[0].id +
+                        '" name="' + replaced[0].name + '" ' + disabled + '>' +
+                        replaced[0].innerHTML +
+                        '</select>',
+                        '<span class="' + variables.customClass + ' ' + statuses.join(' ') + '" aria-hidden="true">',
+                        '<span class="' + variables.valueClass + '">' +
+                            CustomSelectReplacement.setInitialText($el) +
+                        '</span>',
+                        '<span class="icon-fallback-glyph">',
+                        '<span class="icon ' + variables.iconClass + '" aria-hidden="true"></span>',
+                        '<span class="text">Down arrow</span>',
+                        '</span>',
+                        '</span>',
```

Comment:
Indentation is still off? I'd like it to look like [this comment](https://github.com/edx/ux-pattern-library/pull/340/files#r70095659) - I think the ESLint ignores you added should allow that.